ale_linters/swift/sourcekitlsp.vim

```diff
@@ -1,12 +1,25 @@
 " Author: Dan Loman <https://github.com/namolnad>
 " Description: Support for sourcekit-lsp https://github.com/apple/sourcekit-lsp
 
-call ale#Set('sourcekit_lsp_executable', 'sourcekit-lsp')
+let s:default_executable = 'sourcekit-lsp'
+
+call ale#Set('sourcekit_lsp_executable', s:default_executable)
+call ale#Set('swift_sourcekit_lsp_executable', s:default_executable)
+
+function! ale_linters#swift#sourcekitlsp#GetExecutable(buffer) abort
+    if ale#Var(a:buffer, 'sourcekit_lsp_executable') isnot# s:default_executable
+        execute 'echom ''sourcekit_lsp_executable is deprecated. Use `let swift_sourcekitlsp_executable instead`'''
+
+        return ale#Var(a:buffer, 'sourcekit_lsp_executable')
+    endif
+
+    return ale#Var(a:buffer, 'swift_sourcekit_lsp_executable')
+endfunction
 
 call ale#linter#Define('swift', {
 \   'name': 'sourcekitlsp',
```

Comment:
```suggestion
\   'name': 'sourcekit_lsp',
\   'aliases': ['sourcekitlsp'],
```

The variable names should match the linter name, and `sourcekit_lsp` is a better name. Source the codebase for all uses of `sourcekitlsp`, including the name of this file and in the names of functions, and use `sourcekit_lsp` instead.